app/models/defendant.rb

```diff
@@ -75,7 +75,6 @@ def to_builder(hearing: nil)
       defendant.id id
       defendant.prosecutionCaseId prosecution_case_id
       defendant.masterDefendantId masterDefendantId
-      defendant.courtProceedingsInitiated courtProceedingsInitiated.to_datetime if courtProceedingsInitiated.present?
```

Comment:
We need to add:
proceedingsConcluded
isYouth

Reply:
To keep things more straightforward I've not added all the non-required fields. If needed they can be added in the future.